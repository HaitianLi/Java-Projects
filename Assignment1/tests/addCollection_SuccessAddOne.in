add book books.csv 111111
add book books.csv 111113
add book books.csv 111112
add book books.csv 111114
add book books.csv 111115
add book books.csv 111116
add book books.csv 111117
add book books.csv 111118
add book books.csv 111119
add book books.csv 111120
add book books.csv 111121
add book books.csv 111122
add book books.csv 111123
add book books.csv 111124
add book books.csv 111125
add book books.csv 111126
add collection books.csv
exit