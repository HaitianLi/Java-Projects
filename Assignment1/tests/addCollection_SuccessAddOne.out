user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: Successfully added: The Iliad (Homer).

user: Successfully added: The Hobbit (J.R.R. Tolkien).

user: Successfully added: The Odyssey (Homer).

user: Successfully added: The Hitchhiker's Guide to the Galaxy (Douglas Adams).

user: Successfully added: So Long and Thanks for all the Fish (Douglas Adams).

user: Successfully added: Batman Volume 1 (Bob Kane).

user: Successfully added: Harry Potter 1 (J.K. Rowling).

user: Successfully added: Harry Potter 2 (J.K. Rowling).

user: Successfully added: Harry Potter 3 (J.K. Rowling).

user: Successfully added: Harry Potter 4 (J.K. Rowling).

user: Successfully added: Harry Potter 5 (J.K. Rowling).

user: Successfully added: Harry Potter 6 (J.K. Rowling).

user: Successfully added: Wasteland (T.S. Eliot).

user: Successfully added: Biggles in the Battle (W.E. Johns).

user: Successfully added: Aeneid (Virgil).

user: 1 books successfully added.

user: Ending Library process.
